packages/python/test/fixtures/32-fail-duplicate-header/probe.js

```diff
@@ -0,0 +1,18 @@
+const execa = require('execa');
+
+module.exports = async function ({ deploymentUrl, fetch }) {
+  const probeUrl = `https://${deploymentUrl}`;
+  const result = await execa('curl', [
+    probeUrl,
+    '-s',
+    '-H',
+    'foo: bar',
+    '-H',
+    'foo: bar',
```

Comment:
Does the issue only reproduce with `curl` or can you also reproduce with `fetch`?

Reply:
Oh yeah because `fetch` can't set multiple headers, it combines them into a single key with CSV value 🤦

Reply:
Another option is to try with `require('https')` instead of `fetch` because I think that supports multiple headers

Reply:
I tried a couple to no avail, so far. For example `https` requires an object as headers, it then iterates through them and adds the headers in. But you can't specify multiple keys in a object, JS just uses the last one defined. I tried setting it to a list as in `'a': ['a', 'b']` and that didn't fail. So far only `curl` has been able to easily send something like:

```
> user-agent: curl/7.79.1
> accept: */*
> foo: bar
> foo: bar
```

Although technically I could start diving further into one of the http implementations to get it to set headers like that.